pages/partials/attachFilePanel.ejs

```diff
@@ -0,0 +1,99 @@
+<div class="card mb-4" id="attach-file-panel">
+  <div class="card-header bg-secondary text-white">Attached files</div>
+  <table class="table table-sm">
+    <tbody>
+    <% if (file_list.length == 0) { %>
+    <tr><td><i>No attached files</i></td></tr>
+    <% } else { %>
+      <% file_list.forEach(function(file, iFile) { %>
+      <tr>
+        <td style="word-break:break-all;">
+          <i class="far fa-file-alt"></i>
```

Comment:
Icon seems to be superfluous, could probably be removed.

Reply:
I’m also not a huge fan of this icon. However, the reason I put it there is that on the assessment overview page, when a question has attached files, we put the same icon at the end of the question title to indicate that there are attachments. To make it visually clear how that overview-page icon connects to the actual files, I put the same icon next to each file. Without this, I worried that the link wouldn’t be clear. More feedback on this is welcome!

Reply:
What about a paperclip? I think that's a pretty universal symbol for "attachment".

Reply:
Agreed. I switched the icon to a paperclip. We now show a paperclip next to the question title on the assessment overview page, and next to the attached files themselves on the question page. Does this work for you? An alternative would be to put the icon next to the card title on the question page, for example, rather than next to each file.

Reply:
I think It'd prefer it in the card header - I do think that it makes sense to have the icon be *somewhere* when attached files are involved.

Reply:
done